Pipfile

```diff
@@ -10,7 +10,6 @@ flask-sqlalchemy = "==2.3.2"
 psycopg2-binary = ">=2.7.0"
 gunicorn = "==19.8.1"
 celery = "==4.2.0"
-flower = "==0.9.2"
```

Comment:
Could we leave this in for those of us that run flower locally?

Reply:
I'm reluctant to leave this in for two reasons:

1. It changes the meaning of the Pipfile away from it documenting the dependencies of Masu. Masu doesn't depend on Flower directly, which is why this is getting removed.

2. While folks do run some elements locally from time to time, I think doing so has caused certain problems to remain unresolved for far too long. People are avoiding running these tools from within an openshift cluster, which means they avoid seeing certain classes of problem until we try to run the whole app in stg. 

I think it's time to rip that band-aid off and insist that we standardize development on running all components within an openshift cluster, so that we can deal with the problems that crop up head-on.

Reply:
I get the argument of documenting dependencies, but this can easily be a *dev-package*.

Reply:
I can live with that.